src/components/form/WizardRepositoryTable.js

```diff
@@ -71,17 +70,27 @@ const WizardRepositoryTable = (props) => {
     }));
   };
   const locationProp = props?.label?.props?.locationProp;
+  const historyProp = props?.label?.props?.historyProp;
+  const navigateProp = props?.label?.props?.navigateProp;
+  const customReposFleatureFlag = useFeatureFlags(
+    'edge-management.custom_repos_ui'
+  );
   return (
     <>
       {!isLoading && !data?.count > 0 ? (
         <EmptyState
+          target="_blank"
           icon="repository"
           title="No custom repositories available"
           body="Add custom repositories to build RHEL for Edge images with additional packages."
           primaryAction={{
             text: 'Custom repositories',
-            href: paths.repositories,
+            href: customReposFleatureFlag
+              ? '/preview/insights/content'
```

Comment:
can we move to external function and the url for constants?

Reply:
do you mean to move it to CreateImageWizard component? like we have the imageWizardFeatureFlag there?